### PR TITLE
Dlib 19.1 (new formula)

### PR DESCRIPTION
**Pull request by tseemann**
Dlib is a modern C++ toolkit containing machine learning algorithms and tools for creating complex software in C++ to solve real world problems. It is open source software and licensed under the Boost Software License. The introduction contains everything you need to know to get started using the library. However, if after consulting the documentation, you have any questions, comments, or complaints feel free to post in the forums.

### CODE REVIEW

**Comment by dpo** [comment]
https://travis-ci.org/Homebrew/homebrew-science/jobs/119666413#L2497

Seems like this needs `:x11`.

**Comment by tseemann** [comment]
@dpo you're right, it does need X11. Thanks!

File: dlib.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  desc "C++ library for machine learning"
+  homepage "http://dlib.net/"
+
+  url "http://dlib.net/files/dlib-19.1.tar.bz2"
+  sha256 "242f3b8fbc857621d36b5c3f0b32659a9c9e9adccba794cd82d230aa1adb575c"
+
+  depends_on "cmake" => :build
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "openblas"
```
Comment (dpo) [inline dlib.rb]: Does this not build against Accelerate?

**Comment by tseemann** [comment]
@dpo what is Accelerate ?

**Comment by dpo** [comment]
The Accelerate framework is the implementation of BLAS and LAPACK that ships with OSX. You can link it in using `-framework Accelerate` or `-lblas -llapack`. Configure/CMake scripts should detect it. OpenBLAS shouldn't be necessary here, or at least, could be optional.

**Comment by tseemann** [comment]
@dpo I'm not really interested in supporting OS X, so I guess that's why I didn't know this.

**Comment by dpo** [comment]
Ok but the formula should build correctly on OSX.

**Comment by tseemann** [comment]
I'll just get it in and it can be refined for OS X later.

**Comment by ilovezfs** [comment]
```
Warning: Nonstandard bump subject: Dlib 19.1 (new formula)
Warning: Subject should be: dlib 19.1 (new formula)
```

**Comment by tseemann** [comment]
@ilovezfs the pedantry of brew style reaches a new high ;-)

**Comment by ilovezfs** [comment]
I thought you'd appreciate that. :trollface:

**Pull request closed without merging**